10-inference.Rmd

```diff
@@ -0,0 +1,806 @@
+patterns, trends, or relationships in a dataset 
+extend to the wider population are called *inferential questions*. This chapter will start
+with the fundamental ideas of sampling from populations, and then will work towards
+introducing two common techniques in statistical inference: *point estimation* and
+*interval estimation*. 
+
+## Chapter learning objectives 
+By the end of the chapter, students will be able to:
+
+* Describe real-world examples of questions that can be answered with the statistical inference.
+* Define common population parameters (e.g. mean, proportion, standard deviation) that are often estimated using sampled data, and estimate these from a sample.
+* Define the following statistical sampling terms (population, sample, population parameter, point estimate, sampling distribution).
+* Explain the difference between a population parameter and sample point estimate.
+* Use R to draw random samples from a finite population.
+* Use R to create a sampling distribution from a finite population.
+* Describe how sample size influences the sampling distribution.
+* Define bootstrapping.
+* Use R to create a bootstrap distribution to approximate a sampling distribution.
+* Contrast the bootstrap and sampling distributions.
+
+## Why do we need sampling? 
+Statistical inference can help us decide how quantities we observe in
+a subset of data relate to the same quantities in the broader
+population. Here is an example question that we might use statistical inference to answer:
+
+*What proportion of all undergraduate students in North America own an iPhone?*
+
+In the above question, we are interested in making a conclusion about *all*
+undergraduate students in North America.  This is our **population**: 
+in general, the population is the complete collection of individuals or cases we are interested in studying. 
+Further, in the above question, we are interested in computing a quantity&mdash;the proportion
+of iPhone owners&mdash;based on the entire population. This is our **population parameter**:
+in general, a population parameter is a numerical characteristic
+of the entire population. In order to compute this number in the example above, we would need to ask 
+every single undergraduate in North America whether or not they own an iPhone. In practice,
+directly computing population parameters is often time-consuming and costly, and sometimes impossible. 
+
+A more practical approach would be to collect measurements for a **sample**: a subset of
+individuals collected from the population. We can then compute a **sample statistic**&mdash;a numerical
+characteristic of the sample&mdash;that estimates the population parameter. For example, if we 
+randomly selected 100 undergraduate students across North America (the sample) and computed the fraction of those
+students who own an iPhone (the sample statistic), we might suspect that that fraction is a reasonable
+estimate of the full population fraction. 
```

Comment:
We should keep the language consistent here, stick with proportion:

```suggestion
A more practical approach would be to collect measurements for a **sample**: a subset of
individuals collected from the population. We can then compute a **sample statistic**&mdash;a numerical
characteristic of the sample&mdash;that estimates the population parameter. For example, if we 
randomly selected 100 undergraduate students across North America (the sample) and computed the proportion of those
students who own an iPhone (the sample statistic), we might suspect that that proportion is a reasonable
estimate of the proportion of students who own an iPhone in the entire population. 
```

Reply:
Also, do we want to use point estimate in the overview and then switch to sample statistic here?